arangod/Aql/OptimizerRulesFeature.cpp

```diff
@@ -245,6 +245,8 @@ void OptimizerRulesFeature::addRules() {
   registerRule("patch-update-statements", patchUpdateStatementsRule,
                OptimizerRule::patchUpdateStatementsRule_pass9, DoesNotCreateAdditionalPlans, CanBeDisabled);
 
+  registerRule("replace-javascript-funtions", replaceJSFunctions,
```

Comment:
please name the rule something like `replace-aql-functions` - after your PR is through there is no hint left that these aql functions were once implemented in js.